zkvm/src/types.rs

```diff
@@ -303,6 +303,15 @@ impl Value {
     }
 }
 
+impl Expression {
+    pub fn constant(a: Scalar) -> Self {
```

Comment:
if this is generic over `Into<Scalar>`, then we can have simpler one-liners in mintime/maxtime:

```suggestion
    pub fn constant<S: Into<Scalar>>(a: S) -> Self {
```